docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,428 @@
+-  :git-hdl:`AXI_AD7616 <main:library/axi_ad7616>` *
+-  :git-hdl:`AXI_CLKGEN <main:library/axi_clkgen>`
+-  :git-hdl:`AXI_DMAC <main:library/axi_dmac>`
+-  :git-hdl:`AXI_HDMI_TX <main:library/axi_hdmi_tx>`
+-  :git-hdl:`AXI_I2S_ADI <main:library/axi_i2s_adi>`
+-  :git-hdl:`AXI_PWM_GEN <main:library/axi_pwm_gen>`
+-  :git-hdl:`AXI_SPDIF_TX <main:library/axi_spdif_tx>`
+-  :git-hdl:`AXI_SPI_ENGINE <main:library/spi_engine/axi_spi_engine>` **
+-  :git-hdl:`AXI_SYSID <main:library/axi_sysid>`
+-  :git-hdl:`SPI_ENGINE_EXECUTION <main:library/spi_engine/spi_engine_execution>` **
+-  :git-hdl:`SPI_ENGINE_INTERCONNECT <main:library/spi_engine/spi_engine_interconnect>` **
+-  :git-hdl:`SPI_ENGINE_OFFLOAD <main:library/spi_engine/spi_engine_offload>` **
+-  :git-hdl:`SYNC_BITS <main:library/util_cdc/sync_bits.v>`
+-  :git-hdl:`SYSID_ROM <main:library/sysid_rom>`
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+========================  ==========
+Instance                  Address
+========================  ==========
+axi_ad7616_dma            0x44a30000
+ad7616_pwm_gen            0x44b00000
+spi_ad7616_axi_regmap **  0x44a00000
+axi_ad7616 *              0x44a80000
+========================  ==========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` instantiated only for SER_PAR_N=1 (serial interface)
+
+I2C connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - I2C type
+     - I2C manager instance
+     - Alias
+     - Address
+     - I2C subordinate
+   * - PL
+     - iic_fmc
+     - axi_iic_fmc
+     - 0x4162_0000
+     - \-
+   * - PL
+     - iic_main
+     - axi_iic_main
+     - 0x4160_0000
+     - \-
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PL
+     - axi_spi_engine
+     - AD7616
+     - 0
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Software GPIO number is calculated as follows:
+
+-  Zynq-7000: if PS7 is used, then offset is 54
+
+.. list-table::
+   :widths: 25 20 20 20 15
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+   * - adc_reset_n
+     - OUT
+     - 43
+     - 97
+   * - adc_hw_rngsel
+     - OUT
+     - 42:41
+     - 96:95
+   * - adc_os **
+     - IN/OUT/INOUT
```

Comment:
The direction of adc_os should be updated

Reply:
Done.